CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.0.15
```

Comment:
nitpick: since we are adding functionality, I would consider this a minor version bump.

Reply:
yeah - but since this is all BWC I figured it could be a patch bump and be included in all forthcoming releases... WDYT?

Reply:
Major: breaking changes
Minor: new features, backward-compatible
Patch: fixes to existing features. 

As far as Logstash and what we bundle with it is concerned, consuming plugin minor bumps can and should be done in a minor Logstash release.

Reply:
you're right. let's bump minor.